web/src/main/java/uk/ac/ebi/phenotype/chart/AbrChartAndTableProvider.java

```diff
@@ -55,17 +55,14 @@
 	@Autowired
 	ExperimentService es;
 
-    @Autowired
-    private PhenotypePipelineDAO pipelineDAO;
-
 	@Autowired
 	ImpressService impressService;
 
 
 	public String getChart(Integer pipelineId, String acc, List<String> genderList, List<String> zyList,
-			Integer phenotypingCenterId, String strain, String metadataGroup, String alleleAccession, String chartId){
+			String phenotypingCenter, String strain, String metadataGroup, String alleleAccession, String chartId) throws SolrServerException{
 
-    	HashMap<String, ArrayList<UnidimensionalStatsObject>> data = new HashMap(); // <control/experim, ArrayList<dataToPlot>>
+    	HashMap<String, ArrayList<UnidimensionalStatsObject>> data = new HashMap<>(); // <control/experim, ArrayList<dataToPlot>>
```

Comment:
Can this be more generic? i.e.:

Map<String, List<UnidimensionalStatsObject>> data ...